tensorflow/lite/kernels/internal/reference/logistic.h

```diff
@@ -126,6 +126,36 @@ inline void Logistic(const RuntimeShape& input_shape, const int8_t* input_data,
   }
 }
 
+inline void Logistic(const RuntimeShape& input_shape, const uint8_t* input_data,
+                     float input_scale, int input_zero_point,
+                     const RuntimeShape& output_shape, uint8_t* output_data,
+                     float output_scale, int output_zero_point) {
+  const float cutoff_upper = 1.591576576f;
+  const float cutoff_lower = -11.091299057f;
```

Comment:
This function seems almost identical to the `int8_t` version, expect the cutoff values. 
Is it possible to make the code more concise by reusing the code with a template?

Reply:
+1